poker-pals/src/components/HelloWorld.vue

```diff
@@ -23,6 +30,10 @@
       this.socket.on("connected", data => {
         console.log("client received a message: " + data);
       });
+      //Retrieve the current tables available and store into tables array 
+      this.socket.on("serve-tables", rooms => {
```

Comment:
All the additions to this file would likely need to be moved to its own component